packages/lake/__stories__/UploadArea.stories.tsx

```diff
@@ -151,27 +151,28 @@ export const WithSeveralDocuments = ({ layout }: StoryArgs) => {
     {
       id: "1",
       name: "first-document.pdf",
-      finished: true,
+      status: "FINISHED",
```

Comment:
nit: we never use screaming snake case for unions discriminant elsewhere in the codebase (but `camelCase`), so `finished` | `uploading`, etc